learning_journal/templates/create.jinja2

```diff
@@ -4,10 +4,23 @@
             <h3>Create a Post</h3>
             <article class="create">
                 <form method="POST" action="" enctype="multipart/form-data">
-                    <textarea class="blogpost" name="post" cols="80" rows="20"></textarea>
+                    <p>
+                        <label for="title">Title</label>
+                        <br>
```

Comment:
<br> tags are a pox upon HTML.  They should probably be killed and buried forever.  Try using CSS if you need spacing.  (try setting label { display: block } ).